docs/source/configuration.rst

```diff
@@ -251,6 +251,40 @@ implementation of JSON-LD structured data is available for any data provider but
 ``@context``.  Relationships between items can be expressed but is dependent on such relationships being expressed
 by the dataset provider, not pygeoapi.
 
+
+CQL Filter
```

Comment:
What do you think to start a nested section here in the [openapi](https://pygeoapi-cql-filters.readthedocs.io/en/latest/openapi.html) chapter about CQL filters?

Reply:
In  [openapi](https://pygeoapi-cql-filters.readthedocs.io/en/latest/openapi.html) chapter like **using OpenAPI section** I can have another section for **using CQL filters** or even inside  **using OpenAPI section** I can put in Swagger screenshots related to CQL with proper descriptions. I can do that. What do you say?

Reply:
@francbartoli @jorgejesus thoughts on this?

Reply:
@FarheenB @francbartoli  I suggest for the openAPI section to be done as documentation on  GSoC-2, since at the end of GSoC-2 it will be very clear the status of the implementation and we will have (POST for filter-json) and what we can support on filter-text. If you agree I will open a ticket for this, also for  @alex-mathew

Reply:
@jorgejesus yes I think it will be better to add openAPI section at the later phase of GSoC-2 as by then major part of the implementation will be done and documenting it will be easy.

Reply:
@FarheenB @jorgejesus I was thinking about a small piece of docs to avoid forgetting what you have done and the steps for a smooth setup of CQL